polars/polars-algo/src/algo.rs

```diff
@@ -97,7 +97,7 @@ pub fn hist(s: &Series, bins: Option<&Series>, bin_count: Option<usize>) -> Resu
 
     cuts.left_join(&out, [category_str], [category_str])?
         .fill_null(FillNullStrategy::Zero)?
-        .sort(["category"], false)
+        .sort(["category"], false, false)
```

Comment:
Now that I think of this, I think we should make a `struct` called something like `SortOptions` that implements `Default`. That should make calling this from the rust side much more ergonomic.

Reply:
Are you sure that this is something for this PR?

We already have `SortOptions` (s. [here](https://github.com/pola-rs/polars/blob/main/polars/polars-core/src/chunked_array/ops/mod.rs#L478)) and also the possibility to [`sort_with_options`](https://github.com/pola-rs/polars/blob/main/polars/polars-core/src/frame/mod.rs#L1975). The difference between `sort` and `sort_with_options` is that we cannot specify multiple `descending` columns in `sort_with_options` (e.g. [this](https://github.com/pola-rs/polars/pull/9672/files/f91196b80c91f199028064d3ed98475dc53f8c2e#diff-a1f19a667bc11e4c57f3354c86ebdd7957187c45e2ff760abf857b50f8f1033dR857) is not possible with `sort_with_options`). I guess a generalization of `SortOptions` might be needed for this, but I don't see this in this PR.

Reply:
The reason I propose this is because the number of arguments are becoming unwieldy and that can be resolved with an argument struct. Would you be willing to make a follow up PR on that?

Reply:
Yes, already thought about that. I think I might need some guidance on how to best do this design-wise but I can do the implementation.

Reply:
I just created an issue for this: https://github.com/pola-rs/polars/issues/9820